### PR TITLE
Update dg

### PR DESCRIPTION
**Pull request by Gan868611**
# Description

## Type of Change

- [ ] Possible breaking change (Change in feature that may cause existing features to not work correctly, e.g. change in
  schema)
- [x] Non breaking change (Bug fix, New feature)

## What type of PR is this?

- [ ] bug fix
- [ ] cleanup
- [x] documentation
- [ ] feature

## What this PR does / Why we need it:

```docs

```

## Which issue(s) this PR fixes, if:

Fixes #

#### Additional documentation e.g., usage docs, testing docs etc.:

```docs

```

## Checklist

- [x] Changes have been tested
- [x] All automated tests pass
- [x] Relevant Tags and Milestones have been added

### CODE REVIEW

File: docs/images/ExchangeSeqDiagram.png
Comment (Bawfen) [inline docs/images/ExchangeSeqDiagram.png:1]: Just a few minor corrections to suggest here:
The activation bars should be above the dashed lines here
image

The activation bars are missing from the TransactionManager line
Screenshot 2023-04-08 at 11 00 14 PM

**Review by Bawfen — COMMENTED** [review]
Thanks for helping to update the diagrams! Have just commented on a few minor issues I spotted

File: docs/images/ExchangeSeqDiagram.png
Comment (Gan868611) [reply docs/images/ExchangeSeqDiagram.png:1]: This is edited

Comment (Bawfen) [reply docs/images/ExchangeSeqDiagram.png:1]: Looks good, thanks!

**Pull request merged**